src/lists.py

```diff
@@ -7,23 +7,29 @@
 # For the following, DO NOT USE AN ASSIGNMENT (=).
 
 # Change x so that it is [1, 2, 3, 4]
-# YOUR CODE HERE 
+# YOUR CODE HERE
+x.append(4) 
```

Comment:
hope your understanding append and extend well!, as they play an important role later on in the courses